src/idl_gen_julia.cpp

```diff
@@ -0,0 +1,820 @@
+/*
+ * Copyright 2018 Dolby Laboratories. All rights reserved.
```

Comment:
As part of this PR, you need to sign the Google CLA, which as I understand it, means that code that goes into this repo, won't be your copyright anymore.

Reply:
Changed to `Google Inc.`